template/roster-item.hbs

```diff
@@ -4,6 +4,7 @@
    <div class="jsxc-bar__caption jsxc-grow">
       <div title="{{jid}}" class="jsxc-bar__caption__primary">{{name}}</div>
       <div title="{{lastMessage}}" class="jsxc-bar__caption__secondary">{{lastMessage}}</div>
+      <div title="{{tag}}" class="jsxc-bar-tags"></div>
```

Comment:
How does this work with the rename input?

Reply:
![tag1](https://user-images.githubusercontent.com/24775073/103485075-197e5200-4df4-11eb-967e-d64a75048a51.PNG)
renaming of item is still ok